### DISCUSSION TITLE
Date range support for useInput() hook

### DISCUSSION THREAD

**Discussion by seanockert** [gpbl/react-day-picker#1697] [Ideas]
The `useInput()` hook works well to bind a single input to the date picker. However it's not possible to use with a date range, where we have 2 inputs for `from` and `to`. Could we please add this functionality?

**Suggestion**

Could we set the `mode: 'range'` when initialising the props. And have 2 additional inputProps: `inputPropsFrom` and `inputPropsTo` that we can bind to 2 inputs: *to* and *from*

```js
  const { inputPropsFrom, inputPropsTo, dayPickerProps, setSelected } = useInput({
    mode: 'range',
    fromYear: 1900,
    toYear: 2050,
    defaultSelected: new Date()
  })
``` 

`setSelected()` would take an object: `{ from: Date, to: Date }`

If you could point me in the right direction, I'd be happy to investigate this further and make a PR for it.

**Comment by gpbl** [comment]
@seanockert it is indeed a good idea! Not sure what is the current status of `useInput` - it's a while I don't look at it, and we may want to make some improvements there before adding a new feature. What do you think?

**Reply by seanockert** [reply]
Sorry, I haven't had time to look at `useInput` yet but do you have a list of other improvements you'd like to make to it?
